inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp

```diff
@@ -706,7 +706,12 @@ void MKLDNNNormalizeNode::getSupportedDescriptors() {
     weights_prec = tweights->getTensorDesc().getPrecision();
 
     if (weights_prec == Precision::FP32) {
-        weights_blob = tweights;
+        TensorDesc td(Precision::FP32, tweights->getTensorDesc().getDims(), tweights->getTensorDesc().getLayout());
```

Comment:
Why do you need to copy this blob?

Reply:
I just  tried to avoid references from  MKLDNN node to CNNNetwork. On MKLDNNNode cleanup() we releasing reference to CNNNetLayer to free-up CNNNetwork, but this one still keeps it alive.